### PR TITLE
add supported version for prefers-reduced-data

### PR DESCRIPTION
**Pull request by argyleink**
Chromium code merged to enable prototyping with this media query: see https://crbug.com/1051189 status

### CODE REVIEW

**Comment by jpmedley** [comment]
Gang,

Hold off on merging this for a bit.

Joe

**Comment by argyleink** [comment]
Trying to help it be clear that folks can prototype with it behind the experiments flag

**Comment by argyleink** [comment]
todo: add flags array with entry for experimental

**Comment by argyleink** [comment]
cool, i think this is g2g now 👍

File: css/at-rules/media.json [hunk 1]
```diff
@@ -1437,16 +1437,37 @@
...
             "support": {
               "chrome": {
-                "version_added": false,
-                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>."
+                "version_added": "85",
+                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>.",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "enable-experimental-web-platform-features",
```
Comment (ddbeck) [inline css/at-rules/media.json:1445]: There's some back story here in #6009 (which still needs some PRs to formally fix), but I've made a few suggestions to try to be more consistent with other entries like this

```suggestion
                    "name": "#enable-experimental-web-platform-features",
```

File: css/at-rules/media.json [hunk 2]
```diff
@@ -1437,16 +1437,37 @@
...
+                    "value_to_set": "enabled"
```
Comment (ddbeck) [inline css/at-rules/media.json:1446]:
```suggestion
                    "value_to_set": "Enabled"
```

File: css/at-rules/media.json [hunk 3]
```diff
@@ -1437,16 +1437,37 @@
...
               },
               "chrome_android": {
-                "version_added": false,
-                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>."
+                "version_added": "85",
+                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>.",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "enable-experimental-web-platform-features",
```
Comment (ddbeck) [inline css/at-rules/media.json:1456]:
```suggestion
                    "name": "#enable-experimental-web-platform-features",
```

File: css/at-rules/media.json [hunk 4]
```diff
@@ -1437,16 +1437,37 @@
...
               },
               "edge": {
-                "version_added": false,
-                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>."
+                "version_added": "85",
+                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>.",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "enable-experimental-web-platform-features",
```
Comment (ddbeck) [inline css/at-rules/media.json:1467]:
```suggestion
                    "name": "#enable-experimental-web-platform-features",
```

File: css/at-rules/media.json [hunk 5]
```diff
@@ -1437,16 +1437,37 @@
...
+                    "value_to_set": "enabled"
```
Comment (ddbeck) [inline css/at-rules/media.json:1468]:
```suggestion
                    "value_to_set": "Enabled"
```

File: css/at-rules/media.json [hunk 6]
```diff
@@ -1437,16 +1437,37 @@
...
               "chrome_android": {
-                "version_added": false,
-                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>."
+                "version_added": "85",
+                "notes": "See <a href='https://crbug.com/1051189'>bug 1051189</a>.",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "enable-experimental-web-platform-features",
+                    "value_to_set": "enabled"
```
Comment (ddbeck) [inline css/at-rules/media.json:1457]:
```suggestion
                    "value_to_set": "Enabled"
```

**Pull request merged**